synapse/rest/admin/users.py

```diff
@@ -182,6 +183,20 @@ def _filter(a: attr.Attribute) -> bool:
 
         return HTTPStatus.OK, ret
 
+    def _parse_parameter_deactivated(
+        self, request: SynapseRequest
+    ) -> Union[bool, None]:
```

Comment:
`Optional[bool]` is the more idiomatic way to handle a value that could be `None`.

```suggestion
    ) -> Optional[bool]:
```

Likewise below.